From c9607fa276338dbafb3b8b54b0e21a2b8968c37b Mon Sep 17 00:00:00 2001
From: Sasha Levin <sashal@kernel.org>
Date: Wed, 30 Apr 2025 01:36:23 +0100
Subject: virt: acrn: Remove unused list 'acrn_irqfd_clients'

From: Dr. David Alan Gilbert <linux@treblig.org>

[ Upstream commit 97e72c1e0c5424cd8d7b95c31144e713c44c4244 ]

It doesn't look like this was ever used.

Build tested only.

Signed-off-by: "Dr. David Alan Gilbert" <linux@treblig.org>
Acked-by: Fei Li <fei1.li@intel.com>
Link: https://lore.kernel.org/r/20250430003623.313541-1-linux@treblig.org
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Stable-dep-of: 666c7f9e0792 ("virt: acrn: Fix irqfd use-after-free during eventfd shutdown")
Signed-off-by: Sasha Levin <sashal@kernel.org>
---
 drivers/virt/acrn/irqfd.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/virt/acrn/irqfd.c b/drivers/virt/acrn/irqfd.c
index 9994d818bb7e1..5574fbf878209 100644
--- a/drivers/virt/acrn/irqfd.c
+++ b/drivers/virt/acrn/irqfd.c
@@ -16,8 +16,6 @@
 
 #include "acrn_drv.h"
 
-static LIST_HEAD(acrn_irqfd_clients);
-
 /**
  * struct hsm_irqfd - Properties of HSM irqfd
  * @vm:		Associated VM pointer
-- 
2.53.0

